README.md

```diff
@@ -20,10 +20,9 @@ Related community projects:
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
 
 ## Building from source
+ 
+1. Clone or fork this repository
+2. Before opening this project in Visual Studio or VS Code, execute `build.cmd /t:Restore` (Windows) or `./build.sh /t:Restore` (Linux/macOS). This will execute only the part of the build script that downloads and initializes a few required build tools and packages. To run a complete build on command line only, execute `build.cmd` or `build.sh` without arguments.
```

Comment:
Should we just point to the instructions on the Home repo unless repositories have repo specific build instructions? My concern is that if we do change how we build in the future, it would be more work than updating one location. Thoughts?

Reply:
1. We absolutely want some basic, but precise and accurate info in each repo
2. I agree we should wait on the specifics until after we figure out what upcoming changes we want to make to the build system.

Reply:
Yeah let's wait until the build changes are figured out. @pranavkm I think that having a bit of repetition is okay between CONTRIBUTING and build instructions. Though it may be a bit repetitive, I think it's okay.